scripts/dockerhost/start-dockerhosts

```diff
@@ -1,11 +1,40 @@
 #!/bin/bash
 
+set -e
+
 scriptdir=`dirname "$BASH_SOURCE"`
 echo $scriptdir
 
+if ! which nsenter 2>&1 >/dev/null
+then
+  echo "Please install nsenter. It's a part of the util-linux package, which may not be current on your system."
+  exit 1
+fi
+
+if ! sudo ip link | grep -q docker0
+then
```

Comment:
That is correct; the docker0 bridge disappears after executing the make task. I don't know why though.